test/dora/test_dora_fusion.py

```diff
@@ -124,64 +116,59 @@ def test_dora_column_norm(
+        B = torch.randn(K, N, device="cuda", dtype=dtype)
+        C = torch.randn(M, N, device="cuda", dtype=dtype) if epilogue_add else None
+        scale = torch.randn(N, device="cuda", dtype=dtype) if epilogue_scale else None
+
+        D_ref = torch.matmul(A, B)
+        if epilogue_add:
+            D_ref += C
+        if epilogue_scale:
+            D_ref *= scale.unsqueeze(0)
+
+        D_test = triton_mm(A, B, epilogue_source=C, epilogue_scale=scale)
+        check(D_ref, D_test, dtype)
 
 
 MODES = ["default"]
 
-
-@pytest.mark.skip("TODO: torch.compile does not work with custom kernel")
-@pytest.mark.parametrize(
-    "shape, dtype, epilogue_add, epilogue_scale, mode",
-    [[*cfg, mode] for cfg in FUSED_MATMUL_TEST_CONFIGS for mode in MODES][:1],
-    ids=_arg_to_id,
-)
-def test_dora_matmul_compile(shape, dtype, epilogue_add, epilogue_scale, mode):
-    M, K, N = shape
-    A = torch.randn(M, K, device="cuda", dtype=dtype)
-    B = torch.randn(K, N, device="cuda", dtype=dtype)
-    C = torch.randn(M, N, device="cuda", dtype=dtype) if epilogue_add else None
-    scale = torch.randn(N, device="cuda", dtype=dtype) if epilogue_scale else None
-
-    D_ref = torch.matmul(A, B)
-    if epilogue_add:
-        D_ref += C
-    if epilogue_scale:
-        D_ref *= scale.unsqueeze(0)
-
-    D_test = triton_mm(A, B, epilogue_source=C, epilogue_scale=scale)
-    check(D_ref, D_test, dtype)
-
-    triton_compiled = torch.compile(triton_mm, mode=mode)
-    D_compiled = triton_compiled(A, B, epilogue_source=C, epilogue_scale=scale)
-    check(D_ref, D_compiled, dtype)
+class TestDoraMatmulCompile(common_utils.TestCase):
+    @common_utils.parametrize("shape", FUSED_MATMUL_SHAPES[:1])
+    @common_utils.parametrize("dtype", DTYPES)
+    @common_utils.parametrize("epilogue_add", EPILOGUE_ELEMENTWISE_ADD)
+    @common_utils.parametrize("epilogue_scale", EPILOGUE_BROADCAST_SCALE)
+    @common_utils.parametrize("mode", MODES)
+    @unittest.skipIf(not torch.cuda.is_available(), "Need CUDA available")
+    @unittest.skip("TODO: torch.compile does not work with custom kernel")
+    def test_dora_matmul_compile(shape, dtype, epilogue_add, epilogue_scale, mode):
+        M, K, N = shape
+        A = torch.randn(M, K, device="cuda", dtype=dtype)
+        B = torch.randn(K, N, device="cuda", dtype=dtype)
+        C = torch.randn(M, N, device="cuda", dtype=dtype) if epilogue_add else None
+        scale = torch.randn(N, device="cuda", dtype=dtype) if epilogue_scale else None
+
+        D_ref = torch.matmul(A, B)
+        if epilogue_add:
+            D_ref += C
+        if epilogue_scale:
+            D_ref *= scale.unsqueeze(0)
+
+        D_test = triton_mm(A, B, epilogue_source=C, epilogue_scale=scale)
+        check(D_ref, D_test, dtype)
+
+        triton_compiled = torch.compile(triton_mm, mode=mode)
+        D_compiled = triton_compiled(A, B, epilogue_source=C, epilogue_scale=scale)
+        check(D_ref, D_compiled, dtype)
+
```

Comment:
do you need to instantiate? like this: https://github.com/pytorch/ao/blob/47f96f12a4ffa9468f395c667269ca0fa8eef06d/test/dtypes/test_affine_quantized_tensor_parallel.py#L169

Reply:
Oh i dont know how i missed that thank you so much